### DISCUSSION TITLE
HEIMAN HS2WD-E Smart Siren , possible to use it as a repeater?

### DISCUSSION THREAD

**Discussion by hamstie** [Koenkk/zigbee2mqtt#6709] [Devices]
short question, act the  HS2WD-E Smart Siren also as a zigbee repeater (range extender) ?

**Comment by Koenkk** [comment]
Looks like it's a battery powered device, so (probably) no.

**Reply by hamstie** [reply]
it's a wall plug device with backup  battery 

http://www.heimantech.com/product/?type=detail&id=8

**Reply by Koenkk** [reply]
@wojciechczyz is this a end device or router?

**Reply by hamstie** [reply]
got an answer, this device is a router.
https://community.home-assistant.io/t/how-to-get-heiman-siren-with-zha-to-work/180153/36

**Reply by jobsoftinc** [reply]
The one I got functions as a router too.
